app/screens/settings/notification_auto_responder/notification_auto_responder.tsx

```diff
@@ -85,20 +93,68 @@ const NotificationAutoResponder = ({currentUser}: NotificationAutoResponderProps
 
     const styles = getStyleSheet(theme);
 
+    const close = () => popTopScreen(componentId);
+
+    const saveButton = useMemo(() => {
+        return {
+            id: SAVE_OOO_BUTTON_ID,
+            enabled: false,
+            showAsAction: 'always' as const,
+            testID: 'notification_settings.auto_res.save.button',
+            color: theme.sidebarHeaderTextColor,
+            text: intl.formatMessage({id: 'settings.save', defaultMessage: 'Save'}),
+        };
+    }, [theme.sidebarHeaderTextColor]);
+
+    const canSave = useCallback((enabled: boolean) => {
```

Comment:
I think this will work better in an effect, but 0/5

Reply:
corrected